Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/ExternalDB/ExternalDB.tsx

```diff
@@ -100,32 +100,41 @@ export default function ExternalDB(props: { ID: number, Tab: Tab }) {
 
     if (record == null) return null;
     return (
-        <div style={{ width: '100%', height: window.innerHeight - 63, maxHeight: window.innerHeight - 63, overflow: 'hidden', padding: 15 }}>
-            <LoadingScreen Show={requestStatus === 'loading'}/>
+        <div className="container-fluid d-flex h-100 flex-column" style={{ height: 'inherit' }}>
```

Comment:
```suggestion
        <div className="container-fluid d-flex h-100 flex-column">
```
a `d-flex h-100` won't need a `height` property